### PR TITLE
Added category "Windows, Linux, Android and iOS", extra note in, incorrect links in README.md, now the feature request is smaller and TODO.md there are some things from BetterCrewLink

### PR DESCRIPTION
**Pull request by MatadorProBr**
It’s me again and I’ve made some changes, sorry if I’m getting in the way of making a pull request after another but I’m with some creativity to improve README.md and some other things xD

### CODE REVIEW

**Comment by MatadorProBr** [comment]
I recommend that if you are merging this, edit the TODO.md file before and put everything in the correct category because some things I don't know in which category to put 🙂

File: README.md [hunk 1]
```diff
@@ -30,6 +30,8 @@
   
 <b>Notes:</b><br />
 
+- This is a unofficial fork of CrewLink, so any problem, issue or suggestion you have with BetterCrewLink talk to us on our [Discord](https://discord.gg/qDqTzvj4SH) or [GitHub](https://github.com/OhMyGuus/BetterCrewLink/issues), it will not report any problems to Discord nor the official GitHub project of CrewLink they will not support you.
```
Comment (JKohlman) [inline README.md:33]: Should probably read (suggested changes bolded)
"This is **an** unofficial fork of CrewLink, **for** any problem, issue or suggestion you have with BetterCrewLink talk to us on our [Discord](https://discord.gg/qDqTzvj4SH) or [GitHub](https://github.com/OhMyGuus/BetterCrewLink/issues), **do** not report any problems to **the official Discord or GitHub** project of CrewLink **as** they will not support you."

File: README.md [hunk 2]
```diff
@@ -88,17 +94,29 @@ This project implements proximity voice chat in Among Us. Everyone in an Among U
...
+
+A version for [Linux](https://github.com/OhMyGuus/BetterCrewLink/pull/29) is already being developed but nothing has been released yet.
+
+## Android
+
+Download the latest version from [Play Store](https://play.google.com/store/apps/details?id=io.bettercrewlink.app) and run the `BetterCrewlink` application. (requires a PC player)
+
+## iOS
+
+An iOS version has not yet come out but you can use a [web version](https://web.bettercrewl.ink). (requires a PC player)
```
Comment (JKohlman) [inline README.md:111]: Perhaps "the web version" instead of "a web version"

File: TODO.md [hunk 3]
```diff
@@ -15,17 +15,22 @@
...
 
-- [ ] Add a microphone boost slider.
-- [ ] Add a speaker adjustment slider.
-- [ ] Add individual adjustment sliders to each of the players.
+- [x] Add a microphone boost slider.
+- [x] Add a speaker adjustment slider.
+- [x] Add individual adjustment sliders to each of the players.
+- [x] Add a OBS Overlay
+- [x] Add a Android support
+- [ ] Add a iOS Support
```
Comment (JKohlman) [inline TODO.md:23]: Add iOS support

File: TODO.md [hunk 4]
```diff
@@ -15,17 +15,22 @@
...
 ## Voice / WebRTC
 
-- [ ] Add a microphone boost slider.
-- [ ] Add a speaker adjustment slider.
-- [ ] Add individual adjustment sliders to each of the players.
+- [x] Add a microphone boost slider.
+- [x] Add a speaker adjustment slider.
+- [x] Add individual adjustment sliders to each of the players.
+- [x] Add a OBS Overlay
+- [x] Add a Android support
```
Comment (JKohlman) [inline TODO.md:22]: Add Android support

File: TODO.md [hunk 5]
```diff
@@ -15,17 +15,22 @@
...
+- [ ] Add a iOS Support 
+- [x] Add a option in "Host Settings" Can hear through cams.
```
Comment (JKohlman) [inline TODO.md:24]: Add **an** option in "Host Settings" **for hearing** through cams

File: TODO.md [hunk 6]
```diff
@@ -15,17 +15,22 @@
...
+- [x] Add a option in "Host Settings" No talking through walls.
```
Comment (JKohlman) [inline TODO.md:25]: Add **an** option in "Host Settings" **to disable** talking through walls.

File: TODO.md [hunk 7]
```diff
@@ -15,17 +15,22 @@
 
 ## Voice / WebRTC
 
-- [ ] Add a microphone boost slider.
-- [ ] Add a speaker adjustment slider.
-- [ ] Add individual adjustment sliders to each of the players.
+- [x] Add a microphone boost slider.
+- [x] Add a speaker adjustment slider.
+- [x] Add individual adjustment sliders to each of the players.
+- [x] Add a OBS Overlay
```
Comment (JKohlman) [inline TODO.md:21]: Add **an** OBS Overlay

**Review by JKohlman — COMMENTED** [review]
Just some minor things I'd recommend updating. I could have probably done a PR to your branch but this seemed quicker than all the steps involved in that

**Review by MatadorProBr — COMMENTED** [review]
I just saw it and I solved it too, thanks :)

File: TODO.md (hunk 6 above)
Comment (MatadorProBr) [reply TODO.md]: Resolved thanks :)

File: TODO.md (hunk 7 above)
Comment (MatadorProBr) [reply TODO.md]: Resolved thanks :)

File: TODO.md (hunk 5 above)
Comment (MatadorProBr) [reply TODO.md]: Resolved thanks :)

File: TODO.md (hunk 4 above)
Comment (MatadorProBr) [reply TODO.md]: Resolved thanks :)

File: TODO.md (hunk 3 above)
Comment (MatadorProBr) [reply TODO.md]: Resolved thanks :)

File: README.md (hunk 2 above)
Comment (MatadorProBr) [reply README.md]: Resolved thanks :)

File: README.md (hunk 1 above)
Comment (MatadorProBr) [reply README.md]: Resolved thanks :)

**Pull request merged**